SpatialGDK/Documentation/content/generating-a-snapshot.md

```diff
@@ -27,4 +31,9 @@ To generate a snapshot, use the **Snapshot** button on the SpatialOS GDK for Unr
 
  This creates a snapshot called `default.snapshot` in `spatial\snapshots`.
 
-You only need to regenerate snapshots when changing the `SnapshotGenerator.SingletonActorClasses` section in `DefaultEditorSpatialGDK.ini`. For more information, see [Singleton Actors]({{urlRoot}}/content/singleton-actors.md).
+You need to regenerate snapshots when:
+1. Generating schema for a new class.
+1. Modifying replicated properties or RPCs for any class who's schema was previously generated.
```

Comment:
`whose`

Reply:
And also, `RPC signatures`. Not to be confused with the code within RPCs.

Reply:
English is my first language